src/main/java/mate/jdbc/services/DriverService.java

```diff
@@ -0,0 +1,17 @@
+package mate.jdbc.services;
+
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.model.Driver;
+
+public interface DriverService {
+    Driver create(Driver driver);
+
+    Optional<Driver> get(Long id);

```

Comment:
here not allowed Optional
```suggestion
    Driver get(Long id);

```